.circleci/config.yml

```diff
@@ -10,37 +10,49 @@ jobs:
         steps:
             - checkout
 
-            - run:
-                  name: Create virtual env
-                  command: python -m venv || virtualenv venv
-
             - run:
                   name: Write job name
                   command: echo $CIRCLE_JOB > circlejob.txt
 
-            - restore_cache:
-                  key: deps1-{{ .Branch }}-{{ checksum "dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
-
             - run:
                   name: Install dependencies
                   command: |
                       sudo pip install virtualenv --upgrade
+                      python -m venv || virtualenv venv
                       . venv/bin/activate
                       pip install -r dev-requirements.txt
                       npm install --ignore-scripts
 
-            - save_cache:
-                  key: deps1-{{ .Branch }}-{{ checksum "dev-requirements.txt" }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}-{{ checksum ".circleci/config.yml" }}-{{ checksum "circlejob.txt" }}
-                  paths:
-                      - 'venv'
-                      - 'node_modules'
+            - run:
+                name: Install dependencies (dash)
+                command: |
+                    . venv/bin/activate
+                    git clone git@github.com:plotly/dash.git
+                    pip install -e ./dash --quiet
+                    git clone git@github.com:plotly/dash-renderer.git
+                    cd dash-renderer && npm install --ignore-scripts && cd ..
+                    cd dash-renderer && npm run build && cd ..
+                    pip uninstall dash-renderer -y
+                    pip install -e ./dash-renderer --quiet
+                    git clone git@github.com:plotly/dash-html-components.git
+                    cd dash-html-components && npm install --ignore-scripts && cd ..
+                    cd dash-html-components && npm run build && cd ..
+                    pip uninstall dash-html-components -y
+                    pip install -e ./dash-html-components --quiet
+                    git clone git@github.com:plotly/dash-table.git
+                    cd dash-table && npm install --ignore-scripts && cd ..
+                    cd dash-table && npm run build && cd ..
+                    pip uninstall dash-table -y
+                    pip install -e ./dash-table --quiet
```

Comment:
Same as for the renderer's changes in https://github.com/plotly/dash-renderer/pull/130